include/zmq.h

```diff
@@ -371,6 +374,18 @@ ZMQ_EXPORT const char *zmq_msg_gets (const zmq_msg_t *msg, const char *property)
 #define ZMQ_VMCI_BUFFER_MAX_SIZE 87
 #define ZMQ_VMCI_CONNECT_TIMEOUT 88
 #define ZMQ_USE_FD 89
+#define ZMQ_PEER_EVENTS 90
+
+/*  getsockopt additional struct to recover peer events                       */
+typedef struct zmq_gso_peer_events_t
```

Comment:
What does gso mean?

Reply:
GetSockOpt. Trying to shortening struct name.

Reply:
Ah ok :) In particular in the public API, abbreviations should be used reluctantly to avoid confusion. 

I am not sure if it can just be named zmq_peer_events_t, this is even shorter, and maybe sufficiently precise.